osx/Helper/KBLogger.h

```diff
@@ -0,0 +1,11 @@
+//
+//  KBLogger.h
+//  Keybase
+//
+//  Created by Gabriel on 7/22/16.
+//  Copyright © 2016 Keybase. All rights reserved.
```

Comment:
Is this header necessary/correct? Aren't these files covered by the `LICENSE` file at repo root?